crates/cxx-qt-lib/src/core/qtimezone.rs

```diff
@@ -79,6 +111,13 @@ mod ffi {
         #[rust_name = "qtimezone_default"]
         fn qtimezoneDefault() -> UniquePtr<QTimeZone>;
         #[doc(hidden)]
+        #[rust_name = "qtimezone_display_name"]
+        fn qtimezoneDisplayName(
```

Comment:
Is there a reason why this isn't wrapped as a member function?

If not, please replace it with a member function, then you can also remove the added code in the C++ header and source :)
e.g.:
```rust
        fn qtimezoneDisplayName(
            self: &QTimeZone,
            time_type: QTimeZoneTimeType,
            name_type: QTimeZoneNameType,
        ) -> QString;
```

Reply:
It's because the [full method](https://doc.qt.io/qt-6/qtimezone.html#displayName)'s signature uses a `QLocale`, but we don't have a `QLocale`. Which causes this:

```
error: address of overloaded function 'displayName' does not match required type
'QString (QTimeZone::TimeType, QTimeZone::NameType) const'

  ::QString (::QTimeZone::*display_name$)(::QTimeZoneTimeType, ::QTimeZoneNameType) const
  = &::QTimeZone::displayName;

note: candidate function has different number of parameters (expected 2 but has 3)
```

Reply:
Ah, that makes sense.